README.md

```diff
@@ -6,12 +6,13 @@ See documentation [here](doc/00-overview.md)
 
 python:
     reason: 'make sure python flag exists to load config in python.md'
+azure-arm: true
 
 output-folder: $(az-output-folder)
 debug-output-folder: $(az-output-folder)/_az_debug
 
 use-extension:
-  "@autorest/python": "5.0.0-preview.3"
+  "@autorest/python": "5.0.0-preview.7"
   "@autorest/clicommon": "0.4.8"

```

Comment:
@RodgeFu You still need to bump modelerfour version in autorest/clicommon into 4.13.351 and provide a new autorest/clicommon so that they are using the same modelerfour version right?